examples/exercises/plot_iris_exercise.py

```diff
@@ -4,10 +4,6 @@
 ================================
 
 A tutorial exercise for using different SVM kernels.
-
-This exercise is used in the :ref:`using_kernels_tut` part of the
```

Comment:
This links to the old `doc/tutorial/statistical_inference/supervised_learning.rst`, which starts off with:

```rst
Classes are not always linearly separable in feature space. The solution is to
build a decision function that is not linear but may be polynomial instead.
This is done using the *kernel trick* that can be seen as
creating a decision energy by positioning *kernels* on observations:
```


which is nice and explicit, and I don't think the current user guide kernel section does this: https://scikit-learn.org/stable/modules/kernel_approximation.html#kernel-approximation . Would be a separate issue, but we could consider adding bits from this old tutorial page.

Not much text here, I think it is just to show how to use a kernel? I am not sure if there is anywhere in the current user guide that should link to this.